docs/src/extensions.md

```diff
@@ -43,3 +43,10 @@ therefore welcomed.
 
 `cut` can separate fields by whitespace (Space and Tab) with `-w` flag. This
 feature is adopted from [FreeBSD](https://www.freebsd.org/cgi/man.cgi?cut).
+
+## `fmt`
+
+`fmt` has additional flags for prefixes: `-P/--skip-prefix`, `-x/--exact-prefix`, and
+`-X/--exact-skip-prefix`. With `-m/--preserve-headers`, an attempt is made to detect and preserve
+mail headers in the input. `-q/--quick` breaks lines more quickly. And `-T/--tab-width` defines the
+number of spaces representing a tab when determining the line length.
```

Comment:
Do we know whether these options were inspired by other implementations? It might be nice to mention those if there are any.

Reply:
I don't know where the inspiration for those flags came from. The coreutils implementations mentioned in https://github.com/uutils/coreutils/blob/main/CONTRIBUTING.md don't have `fmt` implementations.

Reply:
Hmm indeed. It seems to be added here: https://github.com/uutils/coreutils/pull/272#issuecomment-46516942, but the author seems to have invented them, which is alright.